APE0.rst

```diff
@@ -0,0 +1,393 @@
+  in serving. Candidates can be nominated by anyone, including themselves, and
+  do not have to be `Voting Members <votingmembers>`_ to be nominated.
+  Candidates should provide a brief statement in support of their candidacy.
+* Phase 2, Voting: Each `Voting Member <votingmembers>`_ can vote for zero or
+  more of the candidates, up to the number of candidates. Voting is performed
+  anonymously. Candidates are ranked by the total number of votes they receive.
+  If a tie occurs, it may be resolved by mutual agreement among the candidates,
+  or else the winner will be chosen at random.
+
+The election process is managed by a non-voting Returns Officer nominated by
+the outgoing Coordination Committee, although the process (specific choice of
+web site, etc) can be set up by the Coordination Committee or their delegates.
+For the initial election, the Returns Officer will be nominated by the NumFOCUS
+Executive Officer. Elections will be carried out every year, provided that at
+least one of the Coordination Committee members has reached the end of their
+term or decided to step down.
+
+Term
+^^^^
+Each Coordination Committee member's term runs for three years from when
+the election results are finalized, except when the election was to fill a
+vacancy created partway through a term. In that case, the newly-elected member’s
+term runs for the remainder of the term of the person they are replacing. There
+is no limit to the number of terms that a single individual can be elected for.
+
+For the initial election of Coordination Committee members, two seats will have
+terms of one year, one has a term of two years, and one has a term of three
+years to create a staggered set of replacements and provide continuity in the
+Coordination Committee. The candidate with the largest number of votes gets the
+longest available seat and the smallest the shortest, unless the new committee
+mutually agree to a different allocation.
+
+Vacancies
+^^^^^^^^^
+Coordination Committee members may resign their position at any time.
+
+Whenever there is a vacancy during the regular Coordination Committee term an
+election must take place.
+
+If a Coordination Committee member cannot be contacted for longer than two
+months without prior notification (for example, due to planned leave), then the
+rest of the Coordination Committee may vote to trigger an election to replace
+them for the duration of their term.
+
+Removing Members
+^^^^^^^^^^^^^^^^
+In exceptional circumstances, the `Voting Members <votingmembers>`_ may remove
+one or more sitting Coordination Committee members via a single vote.
+
+Such a vote is triggered when a `Voting Member <votingmembers>`_ calls for one
+publicly on an appropriate Project-wide communication channel, and two other
+active Voting Members second the proposal.
+
+For Coordination Committee Member(s) to be removed, two-thirds of the active
+`Voting Members <votingmembers>`_ must vote in support of the removal.
```

Comment:
```suggestion
For Coordination Committee Member(s) to be removed, two-thirds of the active
`Voting Members <votingmembers>`_ (where the count *includes* the member being
voted on if they are a voting member) must vote in support of the removal.
```
This is a re-do of the suggestion I made in https://github.com/astropy/astropy-APEs/pull/61#discussion_r568184597 but without the conflict